### DISCUSSION TITLE
Welcome to GoGoGo Discussions!

### DISCUSSION THREAD

**Discussion by ZCShou** [ZCShou/GoGoGo#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by asd5421054** [comment]
新版1.13影梭还是会被钉钉检测为虚拟软件，不能正常打卡

**Reply by ZCShou** [reply]
影梭本身就没有适配钉钉打卡，之前能用纯属巧合！这个项目就是为了学习 Android 编程

**Reply by asd5421054** [reply]
好吧，大佬有类似支持钉钉的吗

**Comment by Newbee06** [comment]
为什么微信定位修改不了

**Reply by wardenlzr** [reply]
哈哈，同问

**Reply by Tyrrrrrrr** [reply]
对的，我的微信定位也不行

**Comment by fundgao** [comment]
飞书打卡能用不？

**Comment by mudaoshi** [comment]
搜索时显示”没有匹配结果，请重新搜索“

**Comment by yyds200471** [comment]
看这个搜索显示app服务被禁用怎么回事的

**Comment by JSHSHDJBDB** [comment]
求问：安装完成后，闪退。手机显示影梭屡次停止运行怎么办呀

**Comment by daka126** [comment]
已经更新2.0
https://www.lanzouq.com/yinsuo2
